src/app/AlwaysOn.UI/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN npm install && \
     npm run build
 
 # Create application container
-FROM nginx:alpine
+FROM mcr.microsoft.com/oss/nginx/nginx:1.17.5-alpine
```

Comment:
so far this was using latest. is there a reason why we are going to a specific one? While :latest has its own issues, we otherwise have to keep it up to date. given that we are not using this Dockerfile really anywhere, I'd opt for latest

Reply:
Mainly cause there's no `:latest` - in theory dependabot should take care of it.

Reply:
Besides that, I don't think that we use `latest` anywhere else. 

https://github.com/Azure/Mission-Critical-Online/blob/8b21fa40b97323aa84caf6de978c27e07f4a3a6e/src/infra/monitoring/grafana/Dockerfile#L2

https://github.com/Azure/Mission-Critical-Online/blob/8b21fa40b97323aa84caf6de978c27e07f4a3a6e/src/infra/monitoring/grafana/Dockerfile#L21

https://github.com/Azure/Mission-Critical-Online/blob/8b21fa40b97323aa84caf6de978c27e07f4a3a6e/src/app/AlwaysOn.UI/Dockerfile#L2

So might be a good opportunity to align that.